src/openeo_gfmap/stac/constants.py

```diff
@@ -0,0 +1,232 @@
+"""
+Constants in the STAC collection generated after a series of batch jobs
+"""
+import pystac
+
+TITLE = "WorldCereal Level 1 cache"
+ID = "L1-CACHE"
+DESCRIPTION = "The Level 1 input data cache contains extracted samples of EO data. It's main use is model calibration, allowing faster iterations by providing a cache."
+LICENSE = "CC-BY-4.0"
+LICENSE_LINK = pystac.Link(
+    rel="license",
+    target="https://spdx.org/licenses/CC-BY-4.0.html",
+    media_type=pystac.MediaType.HTML,
+    title="Creative Commons Attribution 4.0 International License",
+)
+STAC_EXTENSIONS = [
+    "https://stac-extensions.github.io/eo/v1.1.0/schema.json",
+    "https://stac-extensions.github.io/file/v2.1.0/schema.json",
+    "https://stac-extensions.github.io/processing/v1.1.0/schema.json",
+    "https://stac-extensions.github.io/projection/v1.1.0/schema.json",
+]
+CONSTELLATION = ["sentinel-2"]
```

Comment:
is this defined at the level where all sensors are aggregated into one STAC collection? Do new sensors have to be added here? What's the typical workflow?

Reply:
To be discussed monday